hpolib/benchmarks/ml/svm_benchmark.py

```diff
@@ -0,0 +1,316 @@
+            ('preprocess_one_hot',
+             ColumnTransformer([
+                 ("categorical", OneHotEncoder(categories=self.categories, sparse=False), self.categorical_data),
+                 ("continuous", "passthrough", ~self.categorical_data)])),
+            ('svm',
+             svm.SVC(gamma=gamma, C=C, random_state=self.rng))
+        ])
+        return model
+
+    @staticmethod
+    def get_configuration_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        """
+        Creates a ConfigSpace.ConfigurationSpace containing all parameters for
+        the SVM Model
+
+        For a detailed explanation of the hyperparameters:
+        https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html
+
+        Parameters
+        ----------
+        seed : int, None
+            Fixing the seed for the ConfigSpace.ConfigurationSpace
+
+        Returns
+        -------
+        ConfigSpace.ConfigurationSpace
+        """
+
+        seed = seed if seed is not None else np.random.randint(1, 100000)
+        cs = CS.ConfigurationSpace(seed=seed)
+
+        cs.add_hyperparameters([
+            CS.UniformFloatHyperparameter('C', lower=-10., upper=10., default_value=0., log=False),
+            CS.UniformFloatHyperparameter('gamma', lower=-10., upper=10., default_value=1., log=False),
+        ])
+        # cs.generate_all_continuous_from_bounds(SupportVectorMachine.get_meta_information()['bounds'])
+        return cs
+
+    @staticmethod
+    def get_fidelity_space(seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        """
+        Creates a ConfigSpace.ConfigurationSpace containing all fidelity parameters for
+        the SupportVector Benchmark
+
+        Fidelities
+        ----------
+        dataset_fraction: float - [0.1, 1]
+            fraction of training data set to use
+
+        Parameters
+        ----------
+        seed : int, None
+            Fixing the seed for the ConfigSpace.ConfigurationSpace
+
+        Returns
+        -------
+        ConfigSpace.ConfigurationSpace
+        """
+        seed = seed if seed is not None else np.random.randint(1, 100000)
+        fidel_space = CS.ConfigurationSpace(seed=seed)
+
+        fidel_space.add_hyperparameters([
+            CS.UniformFloatHyperparameter("dataset_fraction", lower=0.1, upper=1.0, default_value=1.0, log=False),
+        ])
+        return fidel_space
+
+    @staticmethod
+    def get_meta_information():
+        """ Returns the meta information for the benchmark """
+
+        return {'name': 'Support Vector Machine',
+                'references': ["@article{klein-corr16,"
+                               "author = {A. Klein and S. Falkner and S. Bartels and P. Hennig and F. Hutter},"
+                               "title = {Fast Bayesian Optimization of Machine Learning"
+                               "Hyperparameters on Large Datasets},"
+                               "journal = corr,"
```

Comment:
This is out of date. The paper has been published. Please check our publication website (https://ml.informatik.uni-freiburg.de/publications.html) for the papers in 2017 and double-check whether any of these used this non-surrogate based version?